CONTRIBUTING.md

```diff
@@ -62,6 +62,15 @@ out.
 
 ### Deploying
 
+> [!IMPORTANT]  
+> The codebase relies on `solc` v0.8.21. However, note that Solidity v0.8.20
+> introduced a change, making `shanghai` the default EVM version. When deploying
+> on chains that use an older EVM version, ensure you specify the correct EVM
+> version explicitly through the `--evm-version` command flag or the
+> `FOUNDRY_EVM_VERSION` env variable. If you don't do that, there's a high risk
+> that the contracts won't functioning as expected due to the presence of the
```

Comment:
Shouldn't this be `won't function` or `won't be functioning`?

Reply:
Oops you're right, fixed in 61cc09a72d0b6f4e078663264089a16878196782